package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
heroku has strong opinions regarding the location of certain files. We write this top level package.json to leverage client/package.json